terraform/shared/main.tf

```diff
@@ -37,14 +40,15 @@ locals {
 }
 
 module "itse-apps-stage-1" {
-  source                    = "github.com/mozilla-it/terraform-modules//aws/eks?ref=master"
+  source                    = "github.com/mozilla-it/terraform-modules//aws/eks?ref=SE-2169"
```

Comment:
hmm, was this just for testing?  Or do we want to stay pointing at this tag long term?

Reply:
still leaving this here until upstream question gets resolved, but still going to remove it after we decide & merge upstream. just noting here bc you may see it change as we want to test those 3 PRs in terraform-modules.

Reply:
changed, working with new upstream.